js/interface.js

```diff
@@ -315,14 +313,100 @@
     });
   }
 
-  function loadCustomMenuWidgets() {
-    $('.menu-styles-wrapper').addClass('loading');
+  function isNewerVersion(latestVersionMenu, currentMenuVersion) {
+    for (var index = 0; index < latestVersionMenu.length; index++) {
+      if (isNaN(+latestVersionMenu[index])) latestVersionMenu[index] = 0;
+
+      if (isNaN(+currentMenuVersion[index])) currentMenuVersion[index] = 0;
+
+      if (+latestVersionMenu[index] > +currentMenuVersion[index]) return true;
+
+      if (+latestVersionMenu[index] < +currentMenuVersion[index]) return false;
+    }
+
+    return false;
+  }
+
+  /**
+   * Function to filter organizationMenus by version.
+   * @param {Array} data - array of organizationMenus
+   * @returns {Object} latestVersionMenu - object which contains the latest version of current menu package type
+   */
+
+  function getLatestMenuVersion(data) {
+    var latestVersionMenu;
+
+    data.forEach(function(menu) {
+      var formattedMenuVersion = menu.version.split('.');
+      var formattedLatestVersionMenu = latestVersionMenu ? latestVersionMenu.version.split('.') : null;
+
+      if (latestVersionMenu) {
+        latestVersionMenu = isNewerVersion(formattedLatestVersionMenu, formattedMenuVersion) ? latestVersionMenu : menu;
+      }
+
+      if (!latestVersionMenu) {
```

Comment:
Move this to the top of the `forEach` iterator to simplify the code and prevent unnecessary `if` condition.
```suggestion
      if (!latestVersionMenu) {
        latestVersionMenu = menu;
        return;
      }
```